Makefile

```diff
@@ -10,7 +10,7 @@ IMAGE        = origin-cluster-autoscaler-operator
 BUILD_IMAGE ?= registry.ci.openshift.org/openshift/release:golang-1.20
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.31.0
+ENVTEST_K8S_VERSION = 1.32.0
```

Comment:
Tests are passing so this is more of a nit, but are you able to bump this to 1.32.1?

Reply:
done - my mistake :)